v56-team22-surgery-status-board/src/layout/MainLayout.tsx

```diff
@@ -0,0 +1,13 @@
+import Footer from '@/components/Footer';
+import React from 'react';
+
+const MainLayout: React.FC<{ children: React.ReactNode }> = ({ children }) => {
```

Comment:
Btw, any reason why you used React.FC instead of something like 
`type MainLayoutProps = {...}`
and `MainLayout = ({ children } : MainLayoutProps) => ...`

I haven't encountered this pattern before tbh so I'm curious